index.html

```diff
@@ -32,6 +32,16 @@ <h1>I'm Steph, a Junior Software Developer</h1>
 	</section>
```

Comment:
this appears to be a stray closing tag for a section that is no longer there? Or perhaps mislabelled closing tag for the nav?